R/EventStudy.R

```diff
@@ -241,7 +259,7 @@ EventStudy <- function(estimator, data, outcomevar, policyvar, idvar, timevar, c
     if (static) {
         message("post, overidpost, pre, and overidpre are set to 0. A static model will be estimated.")
     } else {
-        data <- ComputeFirstDifferences(data, idvar, timevar, policyvar, timevar_holes)
+        data <- ComputeFirstDifferences(data, idvar, timevar, policyvar, timevar_holes, return_df = FALSE)
```

Comment:
I think we should remove the `return_df` arguments throughout. Most of these functions are internal and, since we only use datatable now, they are not adding anything.